g/go-metrics/go-metrics_3113b8401b8a_ubi_8.5.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : go-metrics
+# Version       : v0.0.0-20181016184325-3113b8401b8a
+# Source repo   : https://github.com/rcrowley/go-metrics
+# Tested on     :  UBI: 8.5
+# Language  : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Muskaan Sheik <Vedang.Wartikar@ibm.com>
```

Comment:
Can we make below change?
```suggestion
# Maintainer    : Muskaan Sheik /  Vedang Wartikar <Vedang.Wartikar@ibm.com>
```

Reply:
I've made the required changes. @Siddhesh-Ghadi @vedangwartikar